tests/api_tests/test_genomic_set_file_handler.py

```diff
@@ -23,7 +23,7 @@
 )
 from rdr_service.model.participant import Participant
 from tests import test_data
-from rdr_service.storage import LocalFilesystemStorageProvider
+from rdr_service.storage import LocalFilesystemStorageProvider  # pylint: disable=unused-import
```

Comment:
@wangy70 any reason to import this specifically here? BaseTestCase uses it internally. I think we can remove this reference.

Reply:
Line removed in next commit.